### DISCUSSION TITLE
Unable to reapply to student pack

### DISCUSSION THREAD

**Discussion by gamemake-eng** [community/community#69833] [GitHub Education]
### Hi everyone,

I was trying to reapply to the student pack after fixing the issue. But, even after logging out and back in it still wont allow me to reapply. And my school email does not allow emails from unverified sources.

**Comment by VedasreeM** [comment]
Here are some steps you can take:

Double-check Billing Information: Ensure that your GitHub billing information matches exactly with what is stated in your academic affiliation document.

Log Out and In Again: Try logging out of your GitHub account, close the browser, and then log in again.

Use a Different Browser or Device: Sometimes, technical issues are browser-specific. Trying a different browser or device might help.

Adjust Email Filters: If your school email is blocking emails from unverified sources, you might need to adjust your email settings to allow messages from GitHub.

Contact GitHub Support: If none of the above steps work, consider reaching out to GitHub support for further assistance.

I hope one of these steps helps resolve the issue. If you have any more questions or encounter any other problems, feel free to let me know. I'm here to help!
